clientApp/src/app/components/supervisor/supervisor-classes-list/hosting-requests-dialog/hosting-requests-dialog.component.html

```diff
@@ -0,0 +1,20 @@
+<h1 mat-dialog-title>Zapytanie o prowadzenie zajęć</h1>
+<div mat-dialog-content>
+    <div *ngIf="!isListEmpty()" >
+        <div *ngFor="let item of requstList" fxLayout="row" fxLayoutAlign="space-between center">
+            <p>{{item[0].user.degree}} {{item[0].user.firstName}} {{item[0].user.lastName}}</p>
```

Comment:
is it possible to get rid of these array indices and use objects properties instead?

Reply:
Refactored